gap/ToolFunctions.gi

```diff
@@ -25,7 +25,7 @@ end );
 
 ##
 InstallGlobalFunction( AutoDoc_WriteDocEntry,
-  function( filestream, list_of_records )
+  function( filestream, list_of_records, maybe_heading... )
```

Comment:
I don't like having `maybe_heading` an optional argument. Just change `maybe_heading...` to `heading`.

Reply:
To be clearer: I suggest to change this to this:
```suggestion
  function( filestream, list_of_records, heading )
```
Then callers should either pass a string for `heading`, or something else, like `fail`, to indicate no heading is desired. That'll make the code a bit simpler, I'd say.